src/main/kotlin/com/hedvig/accountservice/schedulers/domain/commands/BlacklistCommand.kt

```diff
@@ -0,0 +1,9 @@
+package com.hedvig.accountservice.schedulers.domain.commands
+
+import org.axonframework.commandhandling.TargetAggregateIdentifier
+
+data class BlacklistCommand(
```

Comment:
Shall we name this differently?

```suggestion
data class SetBlacklistStateCommand(
```

Reply:
Changed it to "BlacklistSchedulerCommand"

Reply:
👍